requirements.txt

```diff
@@ -2,3 +2,4 @@ jinja2
 pyyaml
 jsonschema
 aiodns
+htmlmin
```

Comment:
We should probably add this to the `default.nix` as well since I use that for deployment & development.